tensorflow/core/kernels/reduction_gpu_kernels.cu.h

```diff
@@ -607,22 +623,22 @@ void LaunchColumnReduction_LTE16Cols(OpKernelContext* ctx, OUT_T out, IN_T in,
                                       TensorShape({static_cast<int64>(
                                           sizeof(T) * extent_y * grid_dim.y)}),
                                       &temp_storage));
-    ColumnReduceMax16ColumnsKernel<<<grid_dim, block_dim, 0, cu_stream>>>(
+    GPU_LAUNCH_KERNEL(ColumnReduceMax16ColumnsKernel, dim3(grid_dim), dim3(block_dim), 0, cu_stream,
         in, (T*)temp_storage.flat<int8_t>().data(), extent_x, extent_y, op,
         init);
 
     dim3 new_grid_dim((grid_dim.y * extent_y + 31) / 32, 1, 1);
```

Comment:
shall the grid dimension calculation take into consideration that AMD wavefront has 64 threads instead of 32?